src/Jonathan_Requirements/Requirements.h

```diff
@@ -0,0 +1,70 @@
+//Terrain Builder
+//Created by Jonathan Morelock 3/12/2021
+#include <GL/glew.h>
+#include <gl/glut.h
+#include <math.h>
+
+#ifndef Requirements_h
+#define Requirements_h
+#define CHUNK_SIZE 32
+#define SNOW 
+/**
+ * \defgroup Terrain Terrain_Mods
+ * @{
+ */
+
+/// Ref: Req. 11 & 12 Using speed (set in main.c "handle_movement" function at line 2441) adjusts the
+/// render_radius (set in main.c line 2715 & modified based on chunk size on line 1628).  
+///
+
+typedef struct {
+    int render_radius;
+} DrawSpeed;
+
+/// Ref: Req. 19 Using x y and z vectors for setting the location of precipitation in the world.
+///
+typedef struct {
+    float x;
+    float y;
+    float z;
+}PrecipitationVectors;
+
+/// Ref: Req. 10 & 19 Using render_radius for Req. 10 to adjust the draw distance during inclimate
```

Comment:
When you use typedef, for example,
typedef struct {
    int render_radius;
} DrawSpeed;
I did search online, I am not sure whether to add a name after typedef struct.
https://www.programiz.com/c-programming/c-structures

typedef struct drawSpeeds {
    int render_radius;
} DrawSpeed;

You use typedef to give a name to your user defined data types. For example, you can use typedef with structure to define a new data type and then use that data type to define structure variables directly.